### ISSUE TITLE
Duplicate entry in the manifest property inside the metadata

### ISSUE THREAD

**Issue by MuthuvelkumarS** [cf-platform-eng/tile-generator#196]
We build the tile and found that duplicate entries are generated in the manifest inside the metadata and that is causing issue while we deploy the tile. if I have tile with the below code
```  
jobs:
  - name: boshhmforwarder
    templates:
    - name: boshhmforwarder
      release: bosh-hm-forwarder
    - name: metron_agent
      release: loggregator
    - name: consul_agent
      release: consul
    dynamic_ip: 1
    properties:
      loggregator:
        tls:
          ca_cert: (( $ops_manager.ca_certificate ))
          metron:
            cert: (( ..cf.doppler.metron_tls_cert.cert_pem ))
            key: (( ..cf.doppler.metron_tls_cert.private_key_pem ))
        etcd:
          require_ssl: true
          machines: (( ..cf.etcd_server.ips ))
          ca_cert: (( $ops_manager.ca_certificate ))
      boshhmforwarder:
        incoming_port: (( .properties.boshhmforwarder_incoming_port.value ))
```

it generates metadata with the below manifest

```
  manifest: |
    boshhmforwarder:
      incoming_port: (( .properties.boshhmforwarder_incoming_port.value ))
    loggregator:
      etcd:
        machines: (( ..cf.etcd_server.ips ))
    loggregator_endpoint:
    boshhmforwarder:
      name: boshhmforwarder
    loggregator:
      name: loggregator
```
it contains the duplicate entries for boshhmforwarder and loggregator. Can you look into this?

**Comment by guidowb** [comment]
[This commit](https://github.com/cf-platform-eng/tile-generator/commit/018bb89fcfc185da5d61b9514049e8a1da5a61e1) causes us to add a section to the manifest for each package in the 'tile.yml' file. If the name of the package collides with a manifest section you provide, you will end up with duplicate keys.

I suppose the best solution is for us to merge the sections into one (assuming individual keys within those sections don't conflict). But because of the way the templating works today, that's not entirely straightforward. Nor is it easy for us to detect this case.

The best I can suggest right now is to change your package names to avoid the conflicts. I think you can do so without any impact other than how you would reference our generated properties (which in this case you probably aren't doing).

Can you try using something like this in your `tile.yml`?

```
packages:
- name: my-loggregator
  type: bosh-release
  path: https://bosh.io/d/github.com/cloudfoundry/loggregator?v=89
- name: my-boshhmforwarder
  type: bosh-release
  path: https://bosh.io/d/github.com/cloudfoundry/bosh-hm-forwarder-release
```

With everything else unchanged?

**Comment by MuthuvelkumarS** [comment]
The workaround with the different package name is working fine. Hope you will get this fixed in your future releases.

**Comment by mboldt** [comment]
Fix going through our CI pipeline now. Hope to have a release out today.

**Comment by mboldt** [comment]
@MuthuvelkumarS, we just released this fix in tile-generator 8.1.3. Please let us know if you try it out.

**Comment by MuthuvelkumarS** [comment]
Yes. it worked now. Thanks.
